spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from spidermon.contrib.actions.templates import ActionWithTemplates
+from spidermon.exceptions import NotConfigured
+
+
+class SendMessage(ActionWithTemplates):
+    title = None
+    fake = False
+
+    def __init__(self,
+                 title=None,
+                 fake=None,
+                 *args, **kwargs):
+        super(SendMessage, self).__init__(*args, **kwargs)
+        self.title = title or self.title
+        self.fake = fake or self.fake
+
+        if not self.title:
+            raise NotConfigured("You must provide a title for the message.")
+
+    @classmethod
+    def from_crawler_kwargs(cls, crawler):
+        return {
+            'title': crawler.settings.get('SPIDERMON_SENTRY_MESSAGE_TITLE'),
+            'fake': crawler.settings.getbool('SPIDERMON_SENTRY_FAKE'),
+        }
+
+    def run_action(self):
+        message = self.get_message()
+        if not self.fake:
+            self.send_message(message)
+        else:
+            print('-'*40)
```

Comment:
Logging with `print` doesn't look good. What do you think include a real logger in this action?

Reply:
Just trying to be consistent what we have for exisitng classes.

Reply:
This is not good in the other classes too, so I believe we could start fixing it in this class now, and then use it as a pattern to fix other classes.

Reply:
+1 for having a logging. Print statements as a log is not a good idea. Let's starting fixing it in everything new we add, latter we can refactory the old code.